openjdk.test.load/src/test.load/net/adoptopenjdk/stf/ClassloadingLoadTest.java

```diff
@@ -71,6 +71,7 @@ public void execute(StfCoreExtension test) throws StfException {
 				.addPrereqJarToClasspath(JavaProcessDefinition.JarId.JUNIT)
 				.addPrereqJarToClasspath(JavaProcessDefinition.JarId.HAMCREST)
 				.addProjectToClasspath("openjdk.test.classloading")
+				.generateCoreDumpAtFirstLoadTestFailure(false)
```

Comment:
Just confirm if the default value is true? Thanks.

Reply:
Thanks for pointing this out and reminding me! By default it is set to false. So we actually do not need this PR. Closing it without delivery.